css/styles.css

```diff
@@ -116,3 +116,248 @@
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: #e5e5e5;
+}
+
+.recent-works {
+  margin: 120px 24px 104px 24px;
+  border-bottom: 1px solid #212121;
+  padding-bottom: 30px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 24px;
+  background-color: #e5e5e5;
+  padding-bottom: 120px;
+}
+
+.grid-item {
+  padding: 20px;
+}
+
+.placeholder-img {
+  height: 300px;
+}
+
+.post-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  margin: 12px 0;
+}
+
+.post-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.buttons {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+}
+
+.buttons li {
+  list-style: none;
+  text-align: center;
+  background-color: #fff;
+  margin-right: 12px;
+  padding: 10px 12px 10px 12px;
+  border: 1px solid #8993a4;
+  cursor: pointer;
+}
+
+.buttons li a {
+  text-decoration: none;
+  color: #091e42;
+}
+
+.see-button li {
+  margin-top: 10px;
+  background: #ff6b00;
+  border: 0;
+  font-weight: 700;
+  font-size: 17px;
+}
+
+.see-button li a {
+  color: #fff;
+}
+
+.art-printing {
+  background-image:
+    linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%),
+    url("../img/works-img/art-printing.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.art-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 60px 0 0 0;
+}
+
+.art-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.button-transparent li {
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.button-transparent li a {
+  color: #fff;
+}
+
+.see-button-2 {
+  background: #ff6b00;
+  margin-top: 22px;
+}
+
+.see-button-2 li {
+  margin-left: 114px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.about {
+  background: #1c1a19;
+}
+
+.about-me {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6f00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.about-desc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 44px 0;
+}
+
+.about-img {
+  background-image: url("../img/about-img/about-bkgrnd.png");
```

Comment:
- [ ] Kindly make sure there are no extra margins in the design. As you can see there is an extra white margin on the right. Which display some content to the right and some to left. Make sure to fix it and align it correctly with correct margins.
- [ ] Kindly fix the background colour as you have noticed its grey background colour in the work section. Make sure to change it to white as per the Figma design.
- Your design:
<img width="361" alt="Screenshot 2022-03-18 at 11 54 31 AM" src="https://user-images.githubusercontent.com/70269507/158971280-dbd86916-2402-40c5-9530-95018349b236.png">
<img width="361" alt="Screenshot 2022-03-18 at 11 54 40 AM" src="https://user-images.githubusercontent.com/70269507/158972233-f87663d3-96fa-4ab2-865e-e477ec7390a4.png">


<img width="361" alt="Screenshot 2022-03-18 at 11 54 50 AM" src="https://user-images.githubusercontent.com/70269507/158971657-465f3a14-fad8-4dd0-bdc6-42cfdc105902.png">
<img width="361" alt="Screenshot 2022-03-18 at 11 54 58 AM" src="https://user-images.githubusercontent.com/70269507/158971921-1f8921aa-bcb2-4315-b3c6-c49dd24f6323.png">
<img width="361" alt="Screenshot 2022-03-18 at 11 55 11 AM" src="https://user-images.githubusercontent.com/70269507/158971948-8d282de0-4198-44a8-a4ca-591f42459dbb.png">